### PR TITLE
push release container images to Docker Hub

### PR DESCRIPTION
**Pull request by dperpeet**
also tag newest images with iso date instead of sequential numbering

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,9 +25,16 @@ release-container: docker-running
 	docker rm -f cockpit-release-stage || true
 	docker run --privileged --name=cockpit-release-stage \
 		--entrypoint=/usr/local/bin/Dockerfile.sh cockpit/infra-release:staged
+	$(eval tag := cockpit/infra-release:$(shell date --iso-8601))
```
Comment (stefwalter) [inline Makefile]: May I suggest moving this line outside of the rule, above the rule and using an upper case variable name? Typically make variables are upper case:

```
TAG := cockpit/infra-release:$(shell date --iso-8601)

release-container:
       ...
```

File: release/release-push [hunk 2]
```diff
@@ -0,0 +1,79 @@
+#!/bin/sh -euf
+#
+# release-push
```
Comment (stefwalter) [inline release/release-push]: The scripts named 'release-xxxx' in this directory are meant to be used during the release process. Lets either rename this script, or move it up a level.

It's a bummer that we need such a complicated script for a simple push  task though. Oh well.

File: Makefile (hunk 1 above)
Comment (dperpeet) [inline Makefile]: I didn't want to declare it global since it's only used there, but sure.

Comment (dperpeet) [inline Makefile]: done

File: release/release-push (hunk 2 above)
Comment (dperpeet) [inline release/release-push]: moved the contents of the script into the Makefile

File: Makefile [hunk 3]
```diff
@@ -20,14 +20,35 @@ release-shell: docker-running
...
+	@true
+
+release-push: docker-running
+	{ \
+	ID=`docker images -q cockpit/infra-release:latest`; \
+	if [ `echo "$$ID" | wc -w` -ne "1" ]; then \
+		echo "Expected exactly one image matching 'cockpit/infra-release:latest'"; \
+		exit 1; \
+	fi; \
+	TAGS=`docker images --format "table {{.Tag}}\t{{.ID}}" | grep $$ID | awk '{print $$1}'`; \
```
Comment (dperpeet) [inline Makefile]: this might seem a bit awkward, but I wanted to make sure that we get a known output format when scraping the image info

**Comment by stefwalter** [comment]
Tested and merged into master.

**Pull request closed without merging**